pkg/diagnose/connectivity/trace_request.go

```diff
@@ -24,10 +24,74 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/config/resolver"
 	"github.com/DataDog/datadog-agent/pkg/config/utils"
+	"github.com/DataDog/datadog-agent/pkg/diagnose/diagnosis"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/datadog-agent/pkg/util/scrubber"
 )
 
+func init() {
+	diagnosis.Register("connectivity-datadog-core-endpoints", diagnose)
+}
+
+func diagnose(diagCfg diagnosis.DiagnoseConfig) []diagnosis.Diagnosis {
+
```

Comment:
```suggestion
```
nit: there's a lot of new lines in this function that I'm not sure are necessary to break apart the functionality of the code. I'd suggest reducing them a bit (e.g. 53, 55, 57 can also probably go)

Reply:
I agree I went bit too far with spaces in this function. Cannot recall what I was thinking. Probably did not clean properly debugging code. Thank you. I have removed few of them.